src/Package/Impl/DataInspect/VisualGrid/SortOrder.cs

```diff
@@ -54,6 +47,10 @@ internal sealed class SortOrder : ISortOrder {
             }
         }
 
+        public void Add(ColumnSortOrder order) {
+            _sortOrderList.Add(order);
```

Comment:
No, since it's purely a UI thing. Sort itself doesn't care (or break) if you sort on the same column twice - it's not particularly useful, but it's not harmful either. And doing nothing is not really a good thing, because if you sort by A, then by B, then by A, it's not the same as sorting by A then by B (which is what happens when the last add is a no-op). It should behave as sorting by B then by A. 

Also, this is basically a lower-level access hatch for the tests currently, to avoid tying it to all the visual stuff - it's not used in product code. In future refactoring, I intend it to make the only way to work with this class, and move the visual-related logic elsewhere, to the UI layer where it belongs.